### PR TITLE
Map Page

### LINKED ISSUE #17: Design Map Component
Map Should have ability to display data points based on chapter locations. 

A modal should appear with chapter contact information on click or hover.

### PR DESCRIPTION
**Pull request by shalarewicz**
Refactors div structure for App.jsx. 
Implements Map Component using Mapbox (#17). Map displays markers for each chapter and chapter contact information in a popup.
MapPage displays a side panel displaying donation needs for the selected chapter
Configures json-server for mocking database calls to /api/chapters and /api/items/
Adds controllers for geocoding chapter locations when adding chapters.

### CODE REVIEW

**Comment by shalarewicz** [comment]
@gkphillips this PR includes changes to the chaptersController to geocode the chapter address when adding a chapter to the database.

**Comment by rbhastie** [comment]
Please remove package.lock.json file

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon server/server.js",
-    "dev": "concurrently 'webpack serve --open' 'nodemon server/server.js'",
+    "dev": "concurrently 'webpack serve --open' 'nodemon server/server.js' ",
```
Comment (rbhastie) [inline package.json:8]: Nitpicking here

```suggestion
    "dev": "concurrently 'webpack serve --open' 'nodemon server/server.js'",
```

**Comment by rbhastie** [comment]
Everything looks great! Will re-review as needed when further changes are made.

File: package.json (hunk 1 above)
Comment (shalarewicz) [reply package.json:8]: made this change locally. Will add to a future commit.

**Review by rbhastie — APPROVED** [review]
Everything looks great!

**Pull request merged**